pulsar-common/src/main/java/com/yahoo/pulsar/common/lookup/data/LookupData.java

```diff
@@ -19,17 +19,29 @@
     private String brokerUrl;
```

Comment:
`LookupData` is the POJO used for the JSON response over HTTP lookup. We shouldn't extend it with other fields that are of no necessary for HTTP lookup